manifests/init.pp

```diff
@@ -114,6 +114,7 @@
     "${lockdir_basepath}/pe_event_forwarding/cache/state",
   ]
 
+  # lint:ignore:140chars
```

Comment:
curious why this didn't trigger before - is this a new rule or something? if it is, wonder if we break the line up and not ignore the rule

Reply:
I think we were using an outdated repo for spec testing. I did flip this though from ignoring it to breaking the line up with a Heredoc. 🙌